### PR TITLE
Local rebuild

### LINKED ISSUE #83: Spin up a Rebuild API pod alongside the deployment to query it

### PR DESCRIPTION
**Pull request by yardenshoham**
Closes #83

### CODE REVIEW

File: upwork-devs/yarden-shoham/local-rebuild/dispatcher/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM python:3-alpine
+WORKDIR /usr/src/app
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+COPY . .
+ENV ENDPOINT=play.min.io \
+    ACCESS_KEY=Q3AM3UQ867SPQQA43P2F \
+    SECRET_KEY=zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG \
```
Comment (DinisCruz) [inline upwork-devs/yarden-shoham/local-rebuild/dispatcher/Dockerfile:8]: Hi, which secret key is this?

File: upwork-devs/yarden-shoham/local-rebuild/dispatcher/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Configuration
+
+Several environment variables may be set for configuration:
+
+| Environment Variable | Description      | Default                                              |
+| -------------------- | ---------------- | ---------------------------------------------------- |
+| `ENDPOINT`           | S3 endpoint      | `play.min.io`                                        |
+| `ACCESS_KEY`         | S3 access key    | `Q3AM3UQ867SPQQA43P2F`                               |
+| `SECRET_KEY`         | S3 secret key    | `zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG`           |
```
Comment (DinisCruz) [inline upwork-devs/yarden-shoham/local-rebuild/dispatcher/README.md:13]: Same here, we can’t put AWS keys on GitHub

File: upwork-devs/yarden-shoham/local-rebuild/downloader/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+WORKDIR /out
+COPY docker-entrypoint.sh custom-entrypoint.sh
+RUN chmod +x custom-entrypoint.sh
+
+FROM minio/mc:RELEASE.2020-09-03T00-08-28Z AS runner
+COPY --from=permissions-giver /out/custom-entrypoint.sh /usr/local/bin
+ENV PROTOCOL=http \
+    ENDPOINT=play.min.io \
+    ACCESS_KEY=Q3AM3UQ867SPQQA43P2F \
+    SECRET_KEY=zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG \
```
Comment (DinisCruz) [inline upwork-devs/yarden-shoham/local-rebuild/downloader/Dockerfile:11]: Another secret key

File: upwork-devs/yarden-shoham/local-rebuild/downloader/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+## Configuration
+
+Several environment variables may be set for configuration:
+
+| Environment Variable | Description                                    | Default                                            |
+| -------------------- | ---------------------------------------------- | -------------------------------------------------- |
+| `PROTOCOL`           | Protocol to use when connecting to S3 endpoint | `http`                                             |
+| `ENDPOINT`           | S3 endpoint                                    | `play.min.io`                                      |
+| `ACCESS_KEY`         | S3 access key                                  | `Q3AM3UQ867SPQQA43P2F`                             |
```
Comment (DinisCruz) [inline upwork-devs/yarden-shoham/local-rebuild/downloader/README.md:13]: And here

**Review by DinisCruz — CHANGES_REQUESTED** [review]
@yardenshoham please remove and invalidate these AWS keys

**Comment by yardenshoham** [comment]
All of these are the default credentials to min.io's default S3. They are not AWS secrets. They're simply demo values. See https://docs.min.io/docs/minio-client-complete-guide.html.

image

> `mc` is pre-configured with https://play.min.io, aliased as "play". It is a hosted MinIO server for testing and development purpose.

**Review by DinisCruz — APPROVED** [review]
Ok, but the security team should pickup these keys

**Pull request merged**